test/services/place.spec.ts

```diff
@@ -135,4 +135,35 @@ describe('services/place.ts', () => {
     expect(actual.contact.type).to.eq(contactType.contact_type);
     expect(actual.contact.contact_type).to.be.undefined;
   });
+
+  it('setPropertiesFromFormData (supports multiple roles)', () => {
+    const contactType = mockSimpleContactType('string', undefined);
+    contactType.user_role = ['role1', 'role2'];
+    contactType.contact_properties = contactType.place_properties;
+    const place = new Place(contactType);
+    place.properties.existing = 'existing';
+
+    const formData = {
+      place_prop: 'abc',
+      contact_prop: 'efg',
+      garbage: 'ghj',
+      user_role: 'role1 role2',
+    };
+    place.setPropertiesFromFormData(formData);
+
+    expect(place.properties).to.deep.eq({
```

Comment:
nit: dunno if these assertions on place.properties are adding value since they are asserted elsewhere .up to you